src/lib/auth.ts

```diff
@@ -1,11 +1,11 @@
-const logger = require('./logger')
-
-const { disableAuth } = require('./../config')
+import logger from './logger'
+import {disableAuth} from './../config'
+import {NextFunction, Request, Response} from "express";
```

Comment:
Need to sort the linter out on this repo but vscode likes quotation marks and semicolons and none of the rest of the code (should) 
```suggestion
import { NextFunction, Request, Response } from 'express'
```

Reply:
Ah this was intellij doing this, will fix

Reply:
fixed